Documentation/About.rst

```diff
@@ -64,14 +26,12 @@ Special thanks to Xavier Perseguers, who wrote some of the original texts
 in the retired TYPO3 Wiki and elsewhere. Some of the texts have been
 incorporated into this document or at least served as a basis.
 
-Further chapters like
-:ref:`rest-common-pitfalls`, :ref:`rest-cheat-sheet`, :ref:`docs-contribute`,
-:ref:`basic-principles`, :ref:`writing-doc-for-ext-start` and :ref:`docs-official-how-you-can-help`
-were added by Sybille Peters.
+Further chapters like were added by Sybille Peters.
```

Comment:
"like were added" sounds like something is missing.

Reply:
thanks for catching this!